src/react-components/src/__tests__/components/DateRangeForm-test.tsx

```diff
@@ -0,0 +1,39 @@
+
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import * as Mocha from "mocha";
+import { stub } from "sinon";
+
+import DateRangeForm from "../../components/01-atoms/Forms/DateRangeForm";
+
+describe("Renders Input", () => {
+  let container;
+  let callback;
+  before(() => {
+    callback = stub();
+    const container = Enzyme.mount(<DateRangeForm
+      formLabel={<>Publication Year</>}
+
+      fromLabelOpts={{ labelContent: <>From</>, id: "FromLabel" }}
+      fromInputOpts={{ inputId: "fromInput" }}
+      fromHelper={{ content: <>EX. 1901</>, id: "fromyearhelper", isError: false }}
+
+      toLabelOpts={{ labelContent: <>To</>, id: "ToLabel" }}
+      toInputOpts={{ inputId: "toInput" }}
+      toHelper={{ content: <>EX. 2000</>, id: "toYearHelper", isError: false }}
+
+      showError={false}
+      error={{ content: <></>, id: "errorId", isError: true }}
+
+      buttonProps={{ id: "submitButtonId", callback: callback, content: <>Submit</> }}
```

Comment:
How do we make buttons `type=submit` now that we're ripped out that assumption? Does that correspond with `type` in the button's prop blob, or is it no longer possible to set a button to be a `submit` button?

Reply:
yeahhh I'm gonna use this opportunity to reverse these prop names since button's going through breaking changes.